code/AssetLib/MDL/MDLLoader.cpp

```diff
@@ -708,6 +708,7 @@ void MDLImporter::InternReadFile_3DGS_MDL345() {
 
                 // read the normal vector from the precalculated normal table
                 MD2::LookupNormalIndex(pcVertices[iIndex].normalIndex, pcMesh->mNormals[iCurrent]);
+                pcMesh->mNormals[iCurrent] *= -1;
                 // pcMesh->mNormals[iCurrent].y *= -1.0f;
```

Comment:
Now that you proved this outcommented, old code is wrong, you can delete it. Otherwise your commit looks a bit like another experiment

Reply:
It was commented out here:
https://github.com/assimp/assimp/commit/c2d888154964d4720940a268e45d3dbb314cba5b#diff-2f88332cbd20db61ae0077d2a0e9238ed20e0fd0496692b7dbf03ce88946a8ebR733
![image](https://user-images.githubusercontent.com/1734117/116670566-8bcd0080-a9a8-11eb-9af2-082b537a1cd8.png)
Possibly whole model orientation is wrong, my commit just fixes normals in the current way, so it's really an experiment.